Opencv-VS/src/img_detection.cpp

```diff
@@ -0,0 +1,123 @@
+#include "img_detection.h"
+
+int detectCircle(cv::Mat &image)
+{
+        cv::Mat imageMask = cv::Mat::zeros(image.size(), image.type());
+        cv::Mat imageDestination = cv::Mat::ones(image.size(), image.type());
+
+        cv::Mat image2 = image.clone();
+
+        GaussianBlur(image, image, cv::Size(5, 5), 0, 0);
+
+        std::vector<cv::Vec3f> circles;
+        HoughCircles(image, circles, cv::HOUGH_GRADIENT, 1, image.rows/4, 50, 130, 0, 0);
+        int numberOfCircles = 0;
+        cvtColor(image, image, cv::COLOR_GRAY2BGR);
+
+        for (size_t i = 0; i < circles.size(); i++) {
+
+
+            cv::Point center(cvRound(circles[i][0]), cvRound(circles[i][1]));
+            int r = cvRound(circles[i][2]);
+            circle(image, center, 7, cv::Scalar(255,0,255), -1, 4, 0);
+            circle(image, center, r, cv::Scalar(0,255,0), 10, 4, 0);
+
+            std::string areaString = std::to_string(r * r);
```

Comment:
naming again: this is actually a simple rule, a variable name should not contain the type of the variable, like here `areaString` we know it is a string, but we don't know any additional information about that area, is it an area of a circle or what?